reconcile/jenkins_job_builds_cleaner.py

```diff
@@ -49,6 +46,8 @@ def find_builds(jenkins, job_names, rules):
                             'rule_keep_hours': rule['keep_hours'],
                             'build_id': build['id'],
                         })
+            # Only act on the first rule matched
+            break
```

Comment:
Shouldn't this be indented a level further inside of (at the end of) `if rule['name_re'].search(job_name)`?